### ISSUE TITLE
Sidebar should stay opened if Tab Center elements have focus

### ISSUE THREAD

**Issue by SoftVision-CiprianMuresan** [bwinton/VerticalTabs#459]
[Affected versions]:
- Firefox 45 and up
- Tab Center 1.22

[Affected Platforms]:
- All Windows
- All Mac OS
- All Linux

[Prerequisites]:
- latest Tab Center add-on is installed

[Steps to reproduce]:
1. Hover over the Tab Center Sidebar and click the "Search" bar.
2. Move the Mouse cursor away from the Sidebar.

[Expected result]:
The Sidebar remains opened.

[Actual result]:
The Sidebar is closed, but the search bar still has focus.

[Notes]:
- The issue is also reproducible if the context menu(right click) is opened for one of the tabs.
- Attached a screen recording of the issue.

[sidebar should stay hidden if TC elements have focus.zip](https://github.com/bwinton/VerticalTabs/files/381506/sidebar.should.stay.hidden.if.TC.elements.have.focus.zip)

**Comment by ericawright** [comment]
Agreed that it should stay open so long as the context menu is open (not just hovered on).
I'm not sure what behavior it should have with search, I believe that if someone mouses off when in the middle of searching they likely want to end their search - perhaps this should clear it and it should lose focus rather than stay open?

**Comment by SoftVision-CiprianMuresan** [comment]
Yes, but what if the user wants to move the cursor away from the text box as he is writing in it? He would be obligated to move the cursor only downwards, because moving away would close the sidebar and he would lose what he wrote up until that point.
In this case your idea wouldn't be a good UX.
